.github/workflows/publish-next.yml

```diff
@@ -157,8 +157,18 @@ jobs:
     secrets:
       NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 
-  build-test:
+  deploy-sample-app:
+    runs-on: ubuntu-latest
+    environment: deployment
     needs: publish
+    steps:
+      - name: Trigger build for beta liveness sample app pointing at next tag
+        run: curl -X POST -d {} $ENDPOINT -H "Content-Type:application/json"
+        env:
+          ENDPOINT: ${{ secrets.LIVENESS_BETA_SAMPLE_APP_BUILD_TRIGGER }}
+
+  build-test:
+    needs: deploy-sample-app
```

Comment:
does it though? can't it run in parallel?

Reply:
Oh I suppose it doesn't, either way the deploy step runs very quickly since it's just firing off a trigger to re-build the sample app, but i can update